src/bundle/Resources/public/scss/_alloyeditor-ez.scss

```diff
@@ -1092,9 +1140,89 @@ pre.ez-has-anchor > .ez-icon {
                 margin: 0.5rem 0;
             }
 
-            .ez-btn-ae--custom-tag {
+            .ez-btn-ae--cancel {
                 color: $ez-ae-color-primary;
-                background-color: $ez-ae-color-secondary;
+                background-color: $ez-color-base-dark;
+                margin-right: calculateRem(5px);
+            }
+
+            .btn-primary {
+                color: $ez-ae-color-primary;
+                margin-left: calculateRem(5px);
+            }
+
+            &--ezyoutube {
+                width: calculateRem(280px);
```

Comment:
Why do you specify width for our custom tags? I think it should be general rule.